### PR TITLE
Ship Component Market Crash (QM rebalance/slight additions)

### PR DESCRIPTION
**Pull request by frawhst**
[BALANCE]
## About the PR 

This reduces the price of cargo bays by a fair margin. They should still be fairly lucrative, but not as absurd as they are currently. To compensate for this, the selling prices of jumpsuits have been roughly doubled, and the prices for food have been bumped up as well. 

In addition, there's a few new items on the generic trader menu. They now buy furniture parts and writing utensils (pens, crayons, etc) and sell gas masks.  To make furniture worth selling I wanted traders to pay a decent price, and to avoid easy flipping I've increased the furniture crate's price somewhat. 

Also includes some pluralization of trader offerings.

## Why's this needed? 
Ship components have been the 'meta' for a while now, and it's a bit stale. The money you get with a ship parts fabricator is so obscene compared to most other QM strategies that there's no reason to do anything else for money (other than mining, but we have miners for that).

The additions to the generic trader list are just to beef up their lists and hopefully provide some more variety. 

## Changelog
```
(u)Asche:
(+)Ship components have fallen in price. On the bright side, space traders are accepting some new items.
```

### CODE REVIEW

**Comment by BatElite** [comment]
Oh no this PR threatens my one way I know how to QM! What ever will I do?
(I'm not really complaining that my method of cheesing is getting bonked. It looks like it'll still be profitable regardless?)

On a more serious note, the only balance thing I want to bring up is with the food price: previously (#1686) produce crates were made more expensive because people would order them and throw them out the sell belt immediately, and IDK if it's a problem that that (but with barcodes) could be sort of profitable again.

**Comment by frawhst** [comment]
> On a more serious note, the only balance thing I want to bring up is with the food price: previously (#1686) produce crates were made more expensive because people would order them and throw them out the sell belt immediately, and IDK if it's a problem that that (but with barcodes) could be sort of profitable again.

That's a good point, and something I hadn't considered. I bumped up the food crate prices to try and avoid this. It's still profitable if you have a food trader giving good prices, but you're limited by the speed of the barcode machine, so you won't be able to do the QM graviton automation shit you were able to pre-produce crate nerf. I also think it's good to encourage people to grow/butcher their own food rather than just ordering tons of crates, so I'm happy with this overall.

I think the dry goods crate was even more profitable before these changes, actually. You could get 2000 credits off a 200 credit crate, even before haggling. Given that I didn't see people abusing it (possibly because of cargo holds overshadowing everything else) I'm not too concerned.

File: code/modules/economy/traders/generic.dm [hunk 1]
```diff
@@ -181,13 +192,29 @@
...
+/datum/commodity/trader/generic/furniture
+	comname = "Furniture Parts"
+	comtype = /obj/item/furniture_parts/
+	price_boundary = list(100, 300)
+	possible_names = list("We need some more furniture to spice up our ship.",
+	"We're building a new lounge, and we need whatever furniture you have.")
+
+/datum/commodity/trader/generic/pens
+	comname = "Writing Utensils"
+	comtype = /obj/item/pen/
```
Comment (ZeWaka) [inline code/modules/economy/traders/generic.dm:217]: not pencils as well?

Comment (frawhst) [reply code/modules/economy/traders/generic.dm:217]: https://github.com/goonstation/goonstation/blob/72a8bd2719fa2cda5533f45f31573842d947ac1a/code/obj/item/pens_writing_etc.dm#L162
uhhh

Comment (ZeWaka) [reply code/modules/economy/traders/generic.dm:217]: lmaooo

**Review by ZeWaka — APPROVED** [review]

**Review by UrsulaMejor — APPROVED** [review]
ok

**Pull request merged**